stack/server.sh

```diff
@@ -61,6 +70,13 @@ then
   indexRAM=$((15 * $totalRAM / 100000))
```

Comment:
TODO: would be nice to check vm memory and set total ram accordingly.  There's been times that we've been burned by this static calculation.

Reply:
Added TODO comment to revisit later